popolo/importers/popolo_json.py

```diff
@@ -573,14 +577,22 @@ def create_identifier(self, popit_collection, popit_id, django_object):
             identifier=popit_id,
         )
 
+    def preserve_related(self, django_main_model, related_object):
```

Comment:
`preserve_related` sounds like an action, I think something more like `should_preserve_related` makes it a little clearer what this is doing.